recipe/meta.yaml

```diff
@@ -16,10 +16,10 @@ build:
 
 requirements:
   host:
-    - python >=3.7
+    - python >=3.8
     - pip
   run:
-    - python >=3.7
+    - python >=3.8
     - packaging
     - tensorflow >=2.8.0
```

Comment:
The new basic requirements are:

- PyTorch >= 2.3.0: AutoKeras is based on Keras. We recommend using the PyTorch backend. Please follow [this pytorch.org-page](https://pytorch.org/get-started/locally/) to install PyTorch. 

   - This means `tensorflow` to be considered to be replaced by [PyTorch](https://github.com/conda-forge/pytorch-cpu-feedstock).


Source: https://autokeras.com/install/